src/components/Book.js

```diff
@@ -21,10 +21,12 @@ const Books = () => {
               id={book.id}
               title={book.title}
               author={book.author}
+              category={book.category}
             />
           ),
         )
       }
+      <div className="horizontal-line" />
```

Comment:
- [ ] @NurkaAmre great work on your design, but some elements are not fitting well, and also the size of the percentage number is too big, kindly check the comparison below:

Zelpin Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/202761339-de3914de-e322-485e-990c-4c566291a5d1.png) | ![image](https://user-images.githubusercontent.com/61408860/202761408-a825bc05-4f04-4d31-856f-88a923a7fc14.png)